tests/system/robot/NVDAInstaller.py

```diff
@@ -0,0 +1,66 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2020-2021 NV Access Limited
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+"""Logic for NVDA installation process tests.
+"""
+
+from robot.libraries.BuiltIn import BuiltIn
+# relative import not used for 'systemTestUtils' because the folder is added to the path for 'libraries'
+# imported methods start with underscore (_) so they don't get imported into robot files as keywords
+from SystemTestSpy import (
+	_getLib,
+)
+
+# Imported for type information
+from robot.libraries.Process import Process as _ProcessLib
+
+from AssertsLib import AssertsLib as _AssertsLib
+
+import NvdaLib as _nvdaLib
+from NvdaLib import NvdaLib as _nvdaRobotLib
+_nvdaProcessAlias = _nvdaRobotLib.nvdaProcessAlias
+
+_builtIn: BuiltIn = BuiltIn()
+_process: _ProcessLib = _getLib("Process")
+_asserts: _AssertsLib = _getLib("AssertsLib")
+
+
+def read_install_dialog():
```

Comment:
Also here, consider adding a docstring to explain the intent of the test:
"Smoke test the launcher dialogs used to install NVDA"